datalad_dataverse/add_sibling_dataverse.py

```diff
@@ -78,18 +67,23 @@ class CreateSiblingDataverse(Interface):
         dict(text="Create a dataverse dataset sibling for sharing and citing",
              code_py="""\
                  > ds = Dataset('.')
-                 > ds.create_sibling_dataverse(url='https://demo.dataverse.org', name='dataverse')
+                 > ds.add_sibling_dataverse(url='https://demo.dataverse.org', name='dataverse')
              """,
              code_cmd="datalad create-sibling-dataverse demo.dataverse.org -s dataverse",
```

Comment:
```suggestion
             code_cmd="datalad add-sibling-dataverse demo.dataverse.org -s dataverse",
```

Reply:
Thx, I will wait a little with these, and see where the API ends up.